components/organisms/TopNav/top-nav.tsx

```diff
@@ -15,31 +16,45 @@ const TopNav: React.FC = () => {
 
   const { data: gitHubUser } = useFetchUser(user?.user_metadata.user_name);
   const userInterest = gitHubUser?.interests.split(",")[0] || "javascript";
+  const router = useRouter();
 
   return (
-
     <header className="top-nav-container flex justify-between items-center z-50 py-0.5 bg-light-slate-3 border-b">
       <div className="flex justify-between items-center mx-auto container px-2 md:px-16">
         <div className="flex gap-3 md:gap-8 items-center flex-wrap ">
           <HeaderLogo withBg={false} textIsBlack />
           {!!user && onboarded ? (
             <>
-              <Link className="text-sm text-light-slate-10" href={"/hub/insights"}>
+              <Link
+                className={`text-sm ${
+                  router.asPath === "/hub/insights" ? "text-light-orange-10" : "text-light-slate-10"
+                }`}
```

Comment:
refactor (non-blocking): Consider using a function that encapsulates whether or not to use `"text-light-orange-10"` or `"text-light-slate-10"`.

```suggestion
                className={`text-sm ${
                  getActiveStyle(router.asPath === "/hub/insights")
                }`}
```

and add a function outside of the component in the file 

```javascript
function getActiveStyle(isActive) {
  return  isActive ? "text-light-orange-10" : "text-light-slate-10"
}
```

Then use this for all the top nav `<Link />` components.

Reply:
Thanks, looks better.